src/p_setup.c

```diff
@@ -577,6 +577,8 @@ void P_LoadSideDefs2(int lump)
 
       sd->textureoffset = SHORT(msd->textureoffset)<<FRACBITS;
       sd->rowoffset = SHORT(msd->rowoffset)<<FRACBITS;
+      // [crispy] smooth texture scrolling
+      sd->basetextureoffset = sd->textureoffset;
```

Comment:
Seems to be missing `sd->baserowoffset = sd->rowoffset;`.

Reply:
Oh, whoops! I accidently attributed this to @kraflab, sorry @rfomin!